client/me/billing-history/transactions-table.jsx

```diff
@@ -136,13 +136,25 @@ var TransactionsTable = React.createClass( {
 		return description;
 	},
 
+	renderPlaceholder() {
+		return (
+			<tr className="billing-history__transaction is-placeholder">
+				<td className="date">
```

Comment:
Looks like `date` is an existing class name, but we may want to consider updating this later to match our css naming conventions.